helpers/image_helper.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'open-uri'
+require_relative '../config/StaticData'
+
+# Class for working with images after convert service
+class ImageHelper
+  def self.save_image_by_link(url)
+    static_dir = StaticData::TMP_DIR + "/#{Time.now.nsec}.png"
```

Comment:
https://ruby-doc.org/stdlib-2.5.3/libdoc/tempfile/rdoc/Tempfile.html
Tempfile should be used for storing tem data

Your variant will be not cleared and just clobber your disk